lib/Dialect/Linalg/Transforms/Tiling.cpp

```diff
@@ -226,6 +245,16 @@ mlir::linalg::tileLinalgOp(OpBuilder &b, LinalgOp op,
                  op.getNumWindowLoops() ==
              tileSizes.size() &&
          "expected matching number of tile sizes and loops");
+
+  // If permutation is empty, use the identity. Build the permutation map otherwise.
+  auto invPermutationMap = AffineMap::getMultiDimIdentityMap(tileSizes.size(),ScopedContext::getContext());
```

Comment:
This file needs a run of clang-format.

Reply:
Done, using clang-format-8. Is that alright?

Reply:
We are running the latest from LLVM repository since we depend on LLVM anyway.

Reply:
Thanks. Done